From 6adeb9c18b4e7c7bec2afc0940e9430fa7c3bcc4 Mon Sep 17 00:00:00 2001
From: Haikel Guemar <hguemar@fedoraproject.org>
Date: Mon, 29 Sep 2014 16:52:25 +0200
Subject: [PATCH] Remove runtime dependency on python-pbr

---
 setup.py               | 1 -
 zaqarclient/version.py | 3 +--
 2 files changed, 1 insertion(+), 3 deletions(-)

diff --git a/setup.py b/setup.py
index 7363757..09230a0 100644
--- a/setup.py
+++ b/setup.py
@@ -26,5 +26,4 @@ except ImportError:
     pass
 
 setuptools.setup(
-    setup_requires=['pbr'],
     pbr=True)
diff --git a/zaqarclient/version.py b/zaqarclient/version.py
index da4c4de..64cdbe7 100644
--- a/zaqarclient/version.py
+++ b/zaqarclient/version.py
@@ -25,5 +25,4 @@ try:
 except pkg_resources.DistributionNotFound:
     # No PKG-INFO? We're probably running from a checkout, then. Let pbr do
     # its thing to figure out a version number.
-    import pbr.version
-    version_string = str(pbr.version.VersionInfo('python-zaqarclient'))
+    version_string = "REDHATZAQARCLIENTVERSION"
-- 
2.1.0

